colab.ipynb

```diff
@@ -79,25 +67,25 @@
         "    - Click Create and ignore the confirmation pop-up.\n",
         "\n",
         "  1. [Download CLIENT Credentials File](https://console.developers.google.com/apis/credentials)\n",
-        "    - Find your newly created key under **OAuth 2.0 Client IDs** and click download arrow on the right and select JSON format.\n",
-        "    - Use the **Folder** icon on left to upload the **/content/client.json** file.\n",
+        "    - Find your newly created key under **OAuth 2.0 Client IDs** and click download arrow on the right and select yaml format.\n",
+        "    - Use the **Folder** icon on left to upload the **/content/client.yaml** file.\n",
```

Comment:
Client files are downloaded from the cloud UI as JSON.

Reply:
```suggestion
        "    - Use the **Folder** icon on left to upload the **/content/client.json** file.\n",
```